clusterware-customize/lib/functions/job-queue.functions.sh

```diff
@@ -268,7 +307,7 @@ job_queue_list_jobs_in_queue() {
     job_status="$2"
 
     job_queue_s3cmd_setup
-    s3_prefix=$(job_queue_bucket_path "${queue}" "${job_status}"/ )
+    s3_prefix=$(job_queue_bucket_put_path "${queue}" "${job_status}"/ )
```

Comment:
It should be `put_path` as (I think) we want the `alces customize job-queue` commands to keep their behaviour for managing the jobs for the login node. This should be clearer now though as I've changed the names as you suggested.

We may later want to change this so `alces customize job-queue` can interact with other nodes, but this would be a larger change and we'd need to add an argument to the commands to run things on different nodes, or alternatively only allow adding jobs for the current node. As it is it would seem odd to use `pull_path`/`node_path` when listing jobs as when we put jobs in the queue we always use `put_path`/`master_path`.